docs/team/README.md

```diff
@@ -35,6 +35,8 @@ Members include Project Management Committee members and committers. The List in
 |:---|:--|
 |Sheng Wang| wangsheng |
 |Yuntao Li| liyuntao |
+|Zhenxu Ke| kezhenxu94 |
+|Yuguang Zhao| zhaoyuguang |
```

Comment:
@zhaoyuguang Don't edit other's PR by adding your inform. And the list should be in alphabeta order.

Reply:
OK